axelar-chains-config/info/mainnet.json

```diff
@@ -2501,6 +2501,13 @@
     "rpc": "https://rpc.ankr.com/stellar_soroban",
     "horizonRpc": "https://horizon.stellar.org",
     "networkType": "mainnet",
+    "chainType": "stellar",
+    "decimals": 7,
+    "finality": "1",
+    "contracts": {
+      "AxelarGateway": {}
```

Comment:
I don't like adding an empty contract field. Makes it more confusing when testing if a contract config is present

Reply:
Let's make the config tests more sensible? Empty contract config is pointless, we should allow no contracts to be specified or if it's specified it should be done properly